include/Dialect/QUIR/Transforms/BreakReset.h

```diff
@@ -24,19 +24,28 @@
 #include "mlir/IR/BuiltinOps.h"
 #include "mlir/Pass/Pass.h"
 
+#include "Dialect/QUIR/IR/QUIROps.h"
+
+#include <deque>
+
 namespace mlir::quir {
 
 /// This pass converts input ResetQubitOps to a parameterized number of
 /// measure and the conditionally called x gates, as well as an optional delay.
 struct BreakResetPass
     : public mlir::PassWrapper<BreakResetPass,
                                mlir::OperationPass<mlir::ModuleOp>> {
+  bool PUT_QUANTUM_GATES_INTO_CIRC = false;
```

Comment:
Is there a reason this is all caps instead of the normal CamelCase?
All caps are usually reserved for preprocessor directives (in my experience).